mesop/bin/bin.py

```diff
@@ -30,13 +30,45 @@
 def main(argv: Sequence[str]):
   if len(argv) < 2:
     print(
-      """\u001b[31mERROR: missing command-line argument to Mesop application.\u001b[0m
+      """\u001b[31mERROR: missing command-line argument to Mesop.\u001b[0m
 
-Example command:
+Example run command:
 $\u001b[35m mesop file.py\u001b[0m"""
     )
     sys.exit(1)
 
+  if argv[1] == "init":
```

Comment:
Question about argv and absl flag. Will absl flag automatically remove the known flags from the argv?

Also is there a way to use argparse with absl flag? Could make working with the command line slightly cleaner.

Reply:
yes, argv is the non-flag arguments.

Turns out you can!
https://github.com/abseil/abseil-py/blob/main/absl/flags/argparse_flags.py